packages/admin/cms-admin/src/generator/future/generateFormField.ts

```diff
@@ -220,6 +220,7 @@ export function generateFormField(
                 fullWidth
                 name="${name}"
                 label={<FormattedMessage id="${instanceGqlType}.${name}" defaultMessage="${label}" />}
+                ${!required ? "displayEmpty" : ""}
```

Comment:
Is this even necessary any longer? Isn't this something the select adds by default?

Reply:
No, it is required. The id of the placeholder item is and empty string, so it is not added to the form values. Without `displayEmpty`, the option is not displayed when selected.

Reply:
I meant it's already added in the FinalFormSelect component: https://github.com/vivid-planet/comet/pull/2294/files/14ad7abd6685275016f681680290a1216769e837#diff-ef93179fe4c6d99e9e776fb1e928ac8b5af12c27fa2d2a6ea124e46028fb8b95R109. Shouldn't this be enough?

Reply:
Oh, you are right. Thanks for noticing!